### PR TITLE
Update NU1605.md

### PR DESCRIPTION
**Pull request by JinwookKim**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[6c826ba](https://github.com/JinwookKim/docs.microsoft.com-nuget/commits/6c826ba7e3e818e12dd16f13f6a84ddfd3ed350c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/errors-and-warnings/NU1605.md](https://github.com/JinwookKim/docs.microsoft.com-nuget/blob/patch-1/docs/reference/errors-and-warnings/NU1605.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605?branch=pr-en-us-1937) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003040019002080-1937%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=%2FrMtbOoKFX91GT5fizZyGrESvY7qgCZf%2BMHz8rM2g7Y%3D&st=2020-03-04T00%3A14%3A38Z&se=2020-04-04T00%3A19%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
@JinwookKim - can you tag the associated issue that you are trying to solve?

File: docs/reference/errors-and-warnings/NU1605.md [hunk 1]
```diff
@@ -62,4 +62,8 @@ To work around this issue, add the following PackageReference:
 The mvc project specified a version constraint on a higher version of a package than restore ultimately resolved. That is because of the [nearest-wins](../../concepts/dependency-resolution.md#nearest-wins) rule - when resolving packages, the version of the nearer package in the graph will override that of the distant package with the same ID.
 
 ### Solution
-This specific error (with Microsoft.NETCore.App package) is improved my moving your .NET Core SDK to 2.2.100 or later. Microsoft.NETCore.App is an auto-referenced package that the .NET Core SDK before version 3.0.100 chooses to bring in automatically. Also see [Self-contained deployment runtime roll forward](/dotnet/core/deploying/runtime-patch-selection).
+This specific error (with Microsoft.NETCore.App package) is improved by moving your .NET Core SDK to 2.2.100 or later. Microsoft.NETCore.App is an auto-referenced package that the .NET Core SDK before version 3.0.100 chooses to bring in automatically. Also see [Self-contained deployment runtime roll forward](/dotnet/core/deploying/runtime-patch-selection).  
+
+You can also try to manually update the packages that are giving you the downgrade error.
```
Comment (nkolev92) [inline docs/reference/errors-and-warnings/NU1605.md]: This shouldn't be true.

Packages marked as "autoreferenced" cannot be updated in VS. 
Auto referenced packages should be only be updated with the SDK as there's no guarantee the SDK you have installed will work with a newer version of the auto refed package.

**Comment by karann-msft** [comment]
closing as abandoned.

**Pull request closed without merging**